doc/dev/crimson.rst

```diff
@@ -142,11 +182,17 @@ corresponding test results are highlighted.
 Debugging Crimson
 =================
 
+Debugging with GDB
+------------------
+
+The `tips`_ for debugging Scylla also apply to Crimson.
+
+.. _tips: https://github.com/scylladb/scylla/blob/master/docs/debugging.md#tips-and-tricks
 
-debugging in your workspace
----------------------------
+Decipher Backtrace with addr2line
```

Comment:
How about:

```suggestion
Human-readable backtraces with addr2line
```
?

Reply:
i like the name. but i want to have symmetric titles for the both cases. i just removed the section title of "debugging crashes of teuthology based tests".